### DISCUSSION TITLE
Open multiple Tab Panels beneath each other

### DISCUSSION THREAD

**Discussion by Hr46ph** [mbnuqw/sidebery#1358] [Ideas]
Basically, I would like to use more of this empty space.
![image](https://github.com/mbnuqw/sidebery/assets/6275310/942bc4a0-d5f7-4d5f-937d-8e0998f4bf4b)

Perhaps something like this, my editing skills are limited to lame copy/paste but I think it gets the idea across for allowing multiple tab groups in which we can switch to another groups.
![image](https://github.com/mbnuqw/sidebery/assets/6275310/220f01af-8dd9-4997-83cf-b6819fa7cef4)

Thanks for considering!

**Comment by rutgersc** [comment]
Just had the same thought. This would be so useful.
